src/components/commands/layout.tsx

```
@@ -5,461 +5,679 @@ import { ChevronDown, Search, Menu, X } from 'lucide-react';
+                    </Badge>
+                  ))}
+                  {!expanded && command.arguments.length > 3 && (
+                    <Badge className="bg-secondary/50 text-secondary-foreground cursor-pointer" onClick={() => setExpanded(true)}>
+                      +{command.arguments.length - 3} more
+                    </Badge>
+                  )}
+                </div>
+              </div>
+            )}
+            
+            {botState?.command_permissions && botState.command_permissions[command.qualified_name || command.name] && (
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)